docs/integration.rst

```diff
@@ -398,6 +398,70 @@ AwsFirehoseHook
 
 .. autoclass:: airflow.contrib.hooks.aws_firehose_hook.AwsFirehoseHook
 
+Amazon SageMaker
+''''''''''''''''
+
+For more instructions on using Amazon SageMaker in Airflow, please see `here`_.
```

Comment:
`here` might have conflicts, better to use a different name

Reply:
Good point! Changed 'here' to 'SageMaker Python SDK README'